pages/windows/xcopy.md

```diff
@@ -0,0 +1,35 @@
+# xcopy
+
+> Copy files and directory trees.
+
+- Copy the file(s) to the specified destination:
+
+`xcopy {{path/to/input}} {{path/to/destination}}`
```

Comment:
Can the input be file or folder both ? If so, then it should be path/to/file_or_folder. https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md#token-syntax pt. 3.